src/1_has_value_checker.js

```diff
@@ -1,3 +1,11 @@
-module.exports = function hasValue() {
-  // TODO Implement me.
+
+
+function hasValue(obj, val) {
+  for (let key in obj) {
+    if (obj[key] === val) {
+      return true;
+    } else {return false}
+  }
```

Comment:
można zapisać prościej:
```suggestion
   return obj[key] === val
  }
```